web3-actions/actions/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
are you using any TS features here anyway? If not you can drop the dependency...